capsules/src/ble.rs

```diff
@@ -0,0 +1,287 @@
+//! Radio/BLE Capsule
+//!
+//! The capsule is implemented on top of a virtual timer
+//! in order to send periodic BLE advertisements without blocking
+//! the kernel
+//!
+//! Currently advertisements with name and data configured in
+//! userland are supported.
+//! The name and data are only configured once i.e., by invoking
+//! start_ble_advertisement() from userland and then periodic advertisement
+//! are handled by the capsule on-top of virtual timers.
+//!
+//! The advertisment intervall is configured to every 150ms by sending on the
```

Comment:
you are right but the function will actually set the advertisment interval in the next refactor thanks to to your comment about how to configure Alarm Frequency.